pennylane/optimize/adagrad.py

```diff
@@ -51,7 +52,7 @@ def __init__(self, stepsize=0.01, eps=1e-8):
         self.eps = eps
         self.accumulation = None
 
-    def apply_grad(self, grad, x):
+    def apply_grad(self, grad, args):
```

Comment:
(docstring below needs updating)

Reply:
Think I've changed `x` to `args` everywhere now.